src/main/java/com/exasol/adapter/document/edml/serializer/EdmlSerializer.java

```diff
@@ -21,7 +22,7 @@
  */
 //[impl->dsn~edml-serialization~1]
 public class EdmlSerializer {
-    private static final String SCHEMA = "https://schemas.exasol.com/edml-1.3.0.json";
+    private static final String SCHEMA = "https://schemas.exasol.com/edml-1.4.0.json";
```

Comment:
Duplicate definition, the version is already defined in `EdmlSchemaValidator.MAPPING_LANGUAGE_SCHEMA`. Maybe use the constant defined there.